src/components/calcite-tip-group/calcite-tip-group.scss

```diff
@@ -1,10 +1,10 @@
 :host {
   @extend %component-host;
-  @extend %component-spacing;
+  //FIXME: Convert this to rem?
   --calcite-tip-max-width: 540px;
 }
 
 ::slotted(calcite-tip) {
-  border: none;
+  @apply border-0;
```

Comment:
☝️ should this be `@apply border-none` to be consistent with the previous rule?

Reply:
I accidently pushed the scss file from tip-group, but that one has it's own PR here:
https://github.com/Esri/calcite-components/pull/1773/files

And I made updates based on your feedback.